apps/bestofjs-nextjs/src/lib/search-utils.tsx

```diff
@@ -1,86 +1,97 @@
+const DEBUG_MODE = false; // to show time spent in search functions
```

Comment:
// TODO: add to .env

😄 ?

Reply:
Thank you, I've added an environment variable accessible from the client: https://github.com/bestofjs/bestofjs/pull/254/commits/674b618630860c79545694b3f241c964ba74b35b